c2corg_api/views/document.py

```diff
@@ -433,6 +433,12 @@ def get(self):
         #    # validate permission (authenticated + associated)
         #    # return 403 if not correct
 
+        title = DBSession.query(DocumentLocale.title) \
+            .filter(DocumentLocale.document_id == id) \
+            .filter(DocumentLocale.culture == lang) \
+            .order_by(DocumentLocale.id.desc()) \
```

Comment:
Do you really need this? Isn't there only one version of the document in DocumentLocale?